_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -92,7 +92,7 @@ Two `<title>` elements where the first one describes the content of the document
 
 #### Passed Example 3
 
-Even though the descriptive `<title>` element is not placed within the `<head>` element that is the context the element can be used in [according to the HTML specification](https://html.spec.whatwg.org/#the-title-element), the rule still passes, since the browser fixes it, and it doesn't cause any known accessibility issues.
+This `<title>` element, which is within the `body`, describes the content of the document. Even though it is not placed within the `<head>` element, as expected [according to the HTML specification](https://html.spec.whatwg.org/#the-title-element), the rule still passes because the browser fixes it and it doesn't cause any known accessibility issues.
```

Comment:
I feel that this lengthy description should go in the similar example in "page has title" rule, rather than here.

Reply:
Possibly a separate issue to this PR.

Reply:
Agreed.